tests/rptest/services/openmessaging_benchmark_configs.py

```diff
@@ -91,20 +91,20 @@ def gte(min_val):
 
     def validate_metrics(metrics, validator):
         """ Validates some predefined metrics rules against the metrics data and throws if any of the rules fail."""
-        assert len(metrics) == 1, "Unexpected metrics output {metrics}}"
-        metrics = next(iter(metrics.values()))
+        assert len(validator) > 0, "At least one metric should be validated"
+
         results = []
         kv_str = lambda k, v: f"Metric {k}, value {v}, "
-        count = 0
-        for key in metrics.keys():
-            if key not in validator:
-                continue
+
+        for key in validator.keys():
+            if key not in metrics:
+                assert False, f"Missing requested validator key {key} in metrics"
```

Comment:
nit: this could just be `assert key in metrics, ...` ?

Reply:
Of course yes, fixed.